target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -123,8 +123,9 @@ get_status_led() {
 	pbr-d1)
 		status_led="$board:orange:power"
 		;;
+	f5d8235-v1|\
 	f5d8235-v2)
-		status_led="$board:blue:router"
```

Comment:
Issue with the blue:router is how it is controlled by GPIO. You can select one of two states as far as I can tell. It either blinks (low) or is on (high). Blinking does of course not look nice and it does not match how failsafe selection is indicated. As a result it looks wired.

Both v2 and v3 have the same LEDs as v1 but the DTS file for v2 is incomplete. I will split the configuration but would strongly recommend to not change the current selected indication of blue:router (see above) and blue:wired for failsafe indication as described.